kink/inject.py

```diff
@@ -203,4 +205,12 @@ def _decorator(_service: ServiceDefinition) -> ServiceResult:
     return _decorator(_service)
 
 
-__all__ = ["inject"]
+class _Injectable:
+    def __getitem__(self, item: Type[I]) -> I:
+        return item
+
+
+Injectable: TypeAlias = _Injectable()
```

Comment:
`TypeAlias` is required by mypy, otherwise, it will start complaining that Injectable is not a valid type.